### PR TITLE
feat: change slide tooltip api

### PR DESCRIPTION
**Pull request by yykoypj**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/36609

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | put tooltip relate props into an object for Slider          |
| 🇨🇳 Chinese | 将Slider的tooltip相关属性放到一个对象中          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/config-provider/__tests__/components.test.js [hunk 1]
```diff
@@ -470,9 +470,14 @@ describe('ConfigProvider', () => {
...
-        myProps.tooltipPrefixCls = `${myProps.prefixCls}-tooltip`;
+        return (
+          <Slider
+            tooltip={{ open: true, prefixCls: `${myProps.prefixCls}-tooltip` }}
+            {...myProps}
+          />
+        );
       }
-      return <Slider tooltipVisible {...myProps} />;
+      return <Slider tooltip {...myProps} />;
```
Comment (afc163) [inline components/config-provider/__tests__/components.test.js:480]:
```suggestion
      return <Slider tooltip={{ open:true }} {...myProps} />;
```

?

Comment (yykoypj) [reply components/config-provider/__tests__/components.test.js:480]: 我原本的想法是tooltip支持boolean类型，相当于原来的tooltipVisible
大佬的意思是只需要支持object类型的形式是吗？

Comment (afc163) [reply components/config-provider/__tests__/components.test.js]: 是的，只写 tooltip 语义不明，不应该相当于 tooltipVisible。

**Review by zombieJ — APPROVED** [review]

File: components/slider/__tests__/index.test.tsx [hunk 2]
```diff
@@ -24,7 +24,9 @@ describe('Slider', () => {
   });
 
   it('should show correct placement tooltip when set tooltipPlacement', () => {
-    const { container } = render(<Slider vertical defaultValue={30} tooltipPlacement="left" />);
+    const { container } = render(
```
Comment (zombieJ) [inline components/slider/__tests__/index.test.tsx:27]: 兼容测试还要留一下，我改改。

**Pull request merged**